### ISSUE TITLE
Columns block: Set width

### ISSUE THREAD

**Issue by scruffian** [WordPress/gutenberg#56206]
## What problem does this address?
While it is possible to set the width of an individual column, there's no control for the Columns block itself. This could be useful when trying to create more complex layouts.

## What is your proposed solution?
It should be possible to control the width of the Columns block.

**Comment by richtabor** [comment]
Outside of wide and full, I think it would be inconsistent to have additional width options on just the columns block. If layout is unblocked, min/max widths could be interesting to expose as dimensions on all relative blocks: https://github.com/WordPress/gutenberg/issues/42385

**Comment by richtabor** [comment]
I do however think it’d be nice if the columns block had column width range controls for each column in its inner blocks, so that you could manipulate the widths of the columns within the columns block, all at once - instead of having to click around to find them individually and remember what % you set each to. 

Separate issue?

**Comment by tellthemachines** [comment]
Closing because this can either be solved by wrapping the Columns in another container such as Grid or Row, or by expanding the alignment options which is covered in #27629.
